rotkehlchen/tests/api/test_manually_tracked_balances.py

```diff
@@ -242,6 +247,25 @@ def test_add_and_query_manually_tracked_balances(
             A_RDN.identifier,
         }
 
+    # delete an asset and check that all the entries are returned but the one with the
+    # deleted asset is marked with the flag
+    globaldb.delete_asset_by_identifier(A_ETH.identifier)
+    AssetResolver.clean_memory_cache()
+    with patch.object(globaldb, '_packaged_db_conn', globaldb.conn):
+        response = requests.get(
+            api_url_for(
+                rotkehlchen_api_server,
+                'manuallytrackedbalancesresource',
+            ),
+        )
+
+    result = assert_proper_sync_response_with_result(response)
+    for entry in result['balances']:
+        if entry['asset'] == A_ETH.identifier:
+            assert result['balances'][0]['asset_is_missing'] is True
+        else:
+            assert 'asset_is_missing' not in result
```

Comment:
Shouldn't this be like that?

```suggestion
            assert 'asset_is_missing' not in result['balances'][0]
```